packages/contracts/src/dollar/DebtCouponManager.sol

```diff
@@ -56,13 +57,17 @@ contract DebtCouponManager is ERC165, IERC1155Receiver {
         couponLengthBlocks = _couponLengthBlocks;
     }
 
+    // cspell:disable-next-line
```

Comment:
disabling was given preference whenever the changed would have caused a downstream artifact to change

Reply:
I don't understand your comment.

Reply:
I think I understand now, if for example we're calling a method on the smart contract from the UI, then the UI code must be updated as well to accommodate the changes. Given that this bounty is for the entire repo, spellchecking the UI code is within scope.

Reply:
correct! yeah, anything downstream. if you have any integrations, then they would also have to be notified and updated. if it is not an issue and you guys own everything, then super